Resources/Private/Form/StaticText.html

```diff
@@ -0,0 +1,6 @@
+<f:if condition="{element.label}">
+    <h2>{element.label -> f:translate(id: 'forms.elements.{element.identifier}.label', package: element.renderingOptions.translationPackage)}</h2>
+</f:if>
+<f:if condition="{element.properties.text}">
+    <p{f:if(condition: element.properties.elementClassAttribute, then: ' class="{element.properties.elementClassAttribute}"')}>{element.properties.text -> f:translate(id: 'forms.elements.{element.identifier}.text', package: '{element.renderingOptions.translationPackage}') -> f:format.nl2br() -> f:format.raw()}</p>
```

Comment:
The order is wrong here
`-> f:format.nl2br() -> f:format.raw()` will lead to escaped `<br>` tags.
`f:format.raw() -> f:format.nl2br()`  should work.

Just as inspiration/idea: This could also be fixed in the form package by adding an `allowHtml` property to the `StaticText` field definition..